chat-db.trace.db

```diff
@@ -928,3 +928,189 @@ Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Ale
 	at org.h2.mvstore.FileStore.open(FileStore.java:173)
 	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
 	... 63 more
+2018-06-23 09:38:54 database: flush
```

Comment:
this file should not be in the repo at all, please remove it 

1) remove `chat-db.*` line from `.gitignore`
commit
2) remove this file
commit
3) add `chat-db.*` line to `.gitignore` back
commit

push